sys-cluster/openais/openais-1.1.4-r1.ebuild

```diff
@@ -21,25 +21,31 @@ DEPEND="${RDEPEND}
 DOCS=( "${S}/AUTHORS" "${S}/README.amf" )
 
 src_prepare() {
+	default
+
 	# respect CFLAGS
 	sed -i -e "s/\$OPT_CFLAGS \$GDB_FLAGS//" configure.ac || die
 	# respect LDFLAGS
 	sed -i -e "s/\$(CFLAGS) -shared/\$(CFLAGS) \$(LDFLAGS) -shared/" \
 		services/Makefile.am || die
 	# don't install docs
 	sed -i -e "/^dist_doc/d" Makefile.am || die
+
 	eautoreconf
 }
 
 src_configure() {
 	econf \
-		--disable-dependency-tracking \
-		--docdir=/usr/share/doc/${PF} \
-		--localstatedir=/var
+		--docdir="${EPREFIX}"/usr/share/doc/${PF} \
+		--localstatedir="${EPREFIX}"/var
 }
 
 src_install() {
-	base_src_install
 	rm -rf "${D}"/etc/init.d/openais || die
-	use static-libs || rm -rf "${D}"/usr/$(get_libdir)/*.a || die
+
+	if ! use static-libs; then
+		find "${D}" -name '*.la' -delete || die "Pruning failed"
+	fi
+
+	default
```

Comment:
doing this last is bad, because you dont have anything to delete. Put this where `base_src_install` was